### ISSUE TITLE
Is there any possibility of a group dimmer custom control?

### ISSUE THREAD

**Issue by mihalski** [andrey-git/home-assistant-custom-ui#5]
I have a number of lights which comprise of multiple bulbs such as this:
![img_2887](https://cloud.githubusercontent.com/assets/3755059/24833068/a9eeaf1a-1d01-11e7-9f58-4a947ef42dde.JPG)
Umm.. Don't know why it rotated 90 degrees.. Imagine them hanging off the ceiling rather than a wall :P

![screen shot 2017-04-09 08 48 15 am](https://cloud.githubusercontent.com/assets/3755059/24833061/823725ec-1d01-11e7-8c7d-e9bfc5fb55ac.png)
Is there any possibility of a light group dimming custom control that works in the same manner as the individual light controls do?

**Comment by andrey-git** [comment]
Handing this in Custom UI would be complicated.

* Currently HA only supports custom state-cards, not custom cards.
* The functionality is also not present in the more-info window.
   - What should the non-dimmable switches do when a group slider is moved?
   - What should the slider show if the slights have different value?

I think the best solution for this is a custom backend component.
Or maybe a new light platform that controls and optionally hides other lights.

**Comment by andrey-git** [comment]
Actually I see that more-info already has this:
If all group members are of the same domains it shows controls for the first member and changing them affects all members.

**Comment by joshluster** [comment]
Because the more-info already has this, is this something that can be implemented?

**Comment by andrey-git** [comment]
Like I allow a single slider on a light state-card to control brightness - you would like a single slider on a group state-card to control brightness of all group members?

**Comment by joshluster** [comment]
Correct, I just discovered that I can have lights I have grouped in hue show up as a light instead of a group.. But for rooms that have multiple lights from different sources this option is still available in the more info area.

**Comment by willscottuk** [comment]
Just wanted to say that I'd love to see this as well. While some of my automations change individual bulbs, 95%+ of manual changes would be a room at a time, so my main frontend view has room-by-room groups of lights.

It'd be great to replace the simple switches with dimmers if at all possible please!

(Screenshot for reference - each of those switches represents a group of lights)
<img width="477" alt="screenshot 2018-01-21 10 16 01" src="https://user-images.githubusercontent.com/28684940/35193011-49768278-fe94-11e7-960c-9ecf9622d449.png">

**Comment by andrey-git** [comment]
I might be able to implement this after home-assistant/home-assistant#11548 lands

**Comment by hvbhome** [comment]
@willscottuk hi,
just to be in the loop here, what happens when you click these groups and get a more-info card?
don't you have a slider for the group there?
I was happily surprised to see that happen in my setup 

![schermafbeelding 2018-01-21 om 16 12 34](https://user-images.githubusercontent.com/33354141/35195580-f25d1a4c-fec5-11e7-86e5-bf760e5a02da.jpg)
![schermafbeelding 2018-01-21 om 16 12 41](https://user-images.githubusercontent.com/33354141/35195581-f281c806-fec5-11e7-876b-9d500705b31b.jpg)
